api/core/api.proto

```diff
@@ -13,6 +13,8 @@ service Core {
   rpc StopService (StopServiceRequest) returns (StopServiceReply) {}
   rpc DeployService (DeployServiceRequest) returns (DeployServiceReply) {}
   rpc DeleteService (DeleteServiceRequest) returns (DeleteServiceReply) {}
+  rpc ListServices (ListServicesRequest) returns (ListServicesReply) {}
+  rpc Service (ServiceRequest) returns (ServiceReply) {}
```

Comment:
I suggest to name `Service` by `GetService`. It will be better for the uniformization and explicitly of the API

Reply:
done

Reply:
Don't forget to rename the files `api/core/service.go` and `api/core/service_test.go`.